encled

```diff
@@ -172,6 +174,16 @@ def main(argv):
             print_status(d[0], d[1], d[3])
         return(0)
 
+    if argv[1].upper() == 'GOOD':
+        for d in devlist:
+            if ((d[3][1]).upper()).replace(" ", "") != 'FAULT_ON' and argv[2].upper() != 'off':
```

Comment:
`.upper()` converts to the upper case, and you compare it with 'off' (which is lower case). It's either `.lower()`, or `OFF`.

Reply:
right, fixed by using `.lower()`